pypesto/sample/util.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
```

Comment:
add a "file docstring" at the top of the file, i.e. one line `"""Do this, very shortly"""`

Reply:
@yannikschaelte why? the rest of the files does not have this as far as I can see

Reply:
not even the `util.py` in the profile folder

Reply:
it's good practice to do so + python pep8, should ideally be done everywhere.